Covid Track/Covid Track/ViewModels/Visit History/VisitHistoryViewModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  VisitHistoryViewModel.swift
+//  Covid Track
+//
+//  Created by Samip shah on 28/7/20.
+//  Copyright © 2020 Samip shah. All rights reserved.
+//
+
+import Foundation
+
+class VisitHistoryViewModel {
+  
+  private let visitRepository:VisitRepository
+  
+  var visits:[VisitViewModel] = []
+  
```

Comment:
Why store property is accessible without setter and getter?

Reply:
I have made the visit property private as it needs to accessed inside the class only.